airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -381,6 +381,36 @@ def get_num_queued_tasks():
         )
 
 
+
+seconds_per_unit = {
+    "s": 1, "m": 60, "h": 3600, "d": 86400, "w": 604800
+}
+def convert_to_seconds(s):
+    return int(s[:-1]) * seconds_per_unit[s[-1]] 
+
+def sla_check(sla_interval, sla_time, max_execution_date, cadence):
+    now = pendulum.now("America/Los_Angeles")
+    interval_in_second = convert_to_seconds(interval)
+
+    if sla_time:
+        naive_sla_time = dateparser.parse("today " + sla_time)
+        sla_time = timezone.localize(naive_sla_time)
+    else:
+        # Usually hourly, intraday
+        sla_time = now
+        
+    checkpoint = sla_time.timestamp() - interval_in_second
+    if now > sla_time and max_execution_date.timestamp() < checkpoint:
+        return 1
```

Comment:
personally prefer returning True/False instead of 1/0 if its meant to be a boolean

Reply:
Let me test to see if boolean works.

Reply:
Great, we can use boolean for that.